tensortrade/features/scalers/percent_normalizer.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2019 The TensorTrade Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+import pandas as pd
+import numpy as np
+
+from gym import Space
+from copy import copy
+from typing import Union, List, Tuple, Dict
+
+from tensortrade.features.feature_transformer import FeatureTransformer
+
+
+class PercentChangeNormalizer(FeatureTransformer):
+    """A transformer for normalizing values within a feature pipeline by the column-wise extrema."""
+
+    def __init__(self,
+                 columns: Union[List[str], str, None] = None,
+                 price_column: str = 'open',
```

Comment:
This `price_column` arg needs to be removed.

Reply:
oops. good catch!